eng/pipelines/corefx-base.yml

```
@@ -22,8 +22,8 @@ parameters:
   #     _framework: (netcoreapp, netfx, uap, etc).
   #     _helixQueues: Windows.Amd64 (Only needed if submitToHelix -> true.) -- Queues should be separated by + if multiple.
 
-  # Required: as part of the variables object, the following values need to be passed:
-  #     _outerloop: true | false
+  # Required: empty | innerloop | outerloop | all
```

should empty in the comment instead be ''?
```suggestion
  # Required: '' | innerloop | outerloop | all
```